widgets/src/SpokeSelector.c

```diff
@@ -291,6 +291,8 @@ static void anaconda_spoke_selector_init(AnacondaSpokeSelector *spoke) {
     spoke->priv->title_label = gtk_label_new(NULL);
     gtk_label_set_text_with_mnemonic(GTK_LABEL(spoke->priv->title_label), _(DEFAULT_TITLE));
     gtk_label_set_justify(GTK_LABEL(spoke->priv->title_label), GTK_JUSTIFY_LEFT);
+    gtk_label_set_line_wrap(GTK_LABEL(spoke->priv->title_label), TRUE);
+    gtk_label_set_max_width_chars(GTK_LABEL(spoke->priv->title_label), 2); // don't grow the tile
```

Comment:
This is a very confusing setting. Maybe the width could be set to 25 as the status label below?

Reply:
OK. It does not matter anyway, as I understand it. This isn't a maximum width in terms of absolutes, but maximal width the widget can expand on its own, or "unconditionally request for itself". It can still expand further, but only if there is space in the parent container. That's what happens here.

(FWIW, 25 would be consistent here, while 2 is the width Radek chose recently for other parts of the GUI. That's why I chose that value originally.)

Reply:
Actually, I can't do that, or rather there is no point. I just realized that 25 *characters* is a different width for the heading which used larger text. So the consistency would be in fact an illusion :-/